bundle-workflow/Jenkinsfile

```diff
@@ -87,6 +92,14 @@ pipeline {
                     }
                 }
             }
+            post() {
+                success {
+                    publishNotification(":white_check-mark:", "Successful Build")
+                }
+                failure {
+                    publishNotification(":warning:", "Failed Build")
```

Comment:
warning? not error?

Reply:
I didn't see an icon that I liked in slack for this, ⚠️  : warning: seems to work well enough.   Open to suggestions

Reply:
It's A-OK

Reply:
I prefer ![image](https://user-images.githubusercontent.com/70038446/133333042-98061afa-3e25-4dbe-accc-866724ca7366.png) icon